volatility3/framework/interfaces/__init__.py

```diff
@@ -22,3 +22,14 @@
     symbols,
     automagic,
 )
+
+__all__ = [
```

Comment:
Nor this

Reply:
removed here: https://github.com/volatilityfoundation/volatility3/pull/1364/commits/f2771bce87e80c7a988247f855e5afa117753654